### PR TITLE
Fix order history font-size and contrast

### LINKED ISSUE #19144: Different global font-size on order history page (BO)
#### Describe the bug

Order history page (BO) has defined global font 2px smaller than on other pages. Tested on clean PS installation.

#### Steps to Reproduce

See on the video: https://vimeo.com/418580145/d0a0dbbc4c

#### Additional information

* PrestaShop version: 1.7.7 beta 1
* PHP version: 7.2

### PR DESCRIPTION
**Pull request by davidglezz**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Order history page has different global font-size and color
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #19144
| How to test?  | Go to Order details and check that header font size is the same as other pages.

Before:
Anotación 2020-07-19 182343

After:
Anotación 2020-07-19 190419

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/20233)

### CODE REVIEW

**Review by NeOMakinG — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

**Comment by Progi1984** [comment]
Need an UX validation : please @TristanLDD :)

**Comment by matks** [comment]
UX validation was provided 🎉 

I dont think we need QA validation, do we ? 🤔

**Review by matks — DISMISSED** [review]

**Comment by NeOMakinG** [comment]
@davidglezz there is a conflict on your PR, can you rebase ? we're gonna approve asap after it :)

**Comment by davidglezz** [comment]
Done. Next time I'll try to do a rebase like matks says in https://dev.to/mathieuks/what-it-means-to-rebase-a-pull-request-submitted-on-github-5717

**Review by matthieu-rolland — DISMISSED** [review]

**Review by eternoendless — DISMISSED** [review]
Please rebase and squash your commits, merge commits may introduce merge errors that are invisible in github

**Comment by davidglezz** [comment]
Rebase done! @eternoendless

**Review by NeOMakinG — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by ngodefroy** [comment]
Hello @davidglezz,
I still have the issue:
Capture d’écran 2020-09-03 à 17 09 44


Thank you

**Comment by davidglezz** [comment]
@ngodefroy have you recompiled the assets?

**Comment by ngodefroy** [comment]
> @ngodefroy have you recompiled the assets?

I actually did :s 
I don't see the contrast difference as your screenshot is.

**Comment by davidglezz** [comment]
Maybe an error with docker image. @ngodefroy

Now, you need to install the dependencies manually

```
docker-compose exec prestashop-git bash
cd /var/www/html/themes/classic/_dev
npm run watch
>>> webpack: not found

THEN

npm install
npm run watch

```

fix

**Comment by ngodefroy** [comment]
Hello, @matks  or can somebody from dev team take this PR?
I can't reproduce after fresh install - and after concerting with QA, they told me to avoid adding by myself the following commands lines:

docker-compose exec prestashop-git bash
cd /var/www/html/themes/classic/_dev
npm run watch
>>> webpack: not found

THEN

npm install
npm run watch

Thank you

**Comment by Progi1984** [comment]
Ping @PrestaShop/prestashop-core-developers : cf message from @ngodefroy

**Comment by PierreRambaud** [comment]
> Maybe an error with docker image. @ngodefroy
> 
> Now, you need to install the dependencies manually
> 
> ```
> docker-compose exec prestashop-git bash
> cd /var/www/html/themes/classic/_dev
> npm run watch
> >>> webpack: not found
> 
> THEN
> 
> npm install
> npm run watch
> ```
> 
> fix

Dependencies are already installed manually since tests by QA are from source and not from a zip :sweat_smile: 

@ngodefroy You must recompile assets after having merged this pull request. Did you do that?

**Comment by ngodefroy** [comment]
Did a fresh install, recompiled assets, I get it now ! 
Capture d’écran 2020-09-22 à 17 22 26


Thank you @davidglezz & @PierreRambaud :)

**Comment by Progi1984** [comment]
Thanks @davidglezz & @ngodefroy

**Pull request merged**
